test/src/test/java/org/corfudb/recovery/FastObjectLoaderTest.java

```diff
@@ -594,26 +594,6 @@ public void failBecauseOfTrimAnd1Attempt() throws Exception {
 
     }
 
-    @Test
-    public void failWhenTrimHappensWhileFastLoading() throws Exception{
```

Comment:
Don't remove this test. Please replace it with this test:
https://github.com/CorfuDB/CorfuDB/pull/1819/files#diff-c19ccef54c8e5dfbb5cd99aa987eeee8L595

Reply:
done